circuit_tutorials/circom/sudoku/README.md

```diff
@@ -1,7 +1,10 @@
 # Sudoku Circuit
 
-This circuit accepts a (public) sudoku puzzle and a (private) solution to that puzzle. Both are formatted as 81-long integer lists with entries 1-9.  
+This circuit accepts a (public) sudoku puzzle and a (private) solution to that puzzle; it has 11906 constraints.
+Both are formatted as 81-long integer lists with entries 1-9.  
```

Comment:
This line `Both are formatted as 81-long integer lists with entries 1-9.` is not clear. If this is describing something like this:
```javascript
exports.p = Scalar.fromString(
  "21888242871839275222246405745257275088548364400416034343698204186575808495617"
);
```
then I'd suggest `Both are formatted as 81 digit-long strings made up of the digits "0" - "9".`